website/static/css/team.css

```diff
@@ -0,0 +1,137 @@
+.page_header {

```

Comment:
Looks like there are similar styles from video.css https://github.com/babel/website/pull/1524/files#r164283741

maybe should move to another css like 'page' or to the babel.css file? Not sure the best way

Reply:
Yeah. We can have it in babel.css file. I feel we should create `Header` component, and re-use them on both videos and team page.

Reply:
I have shifted common styles from video.css & team.css to babel.css